File: python/easy/max_consective_ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums) == 0:
            return 0
        cnt = 0
        max_len = 0
        for num in nums:
            if num == 1:
                cnt += 1
            if num == 0:
                max_len = max(cnt, max_len)
                cnt = 0
        max_len = max(cnt, max_len)
        return max_len

File: cpp/easy/max_consective_ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        if(nums.size() == 0)
            return 0;
        int max_len = 0;
        int len = 0;
        for(int i = 0; i < nums.size(); ++i)
        {
            if(nums[i] == 1)
                ++len;
            if(nums[i] == 0)
            {
                max_len = max(len, max_len);
                len = 0;
            }
        }
        max_len = max(len, max_len);
        return max_len;
    }
};


Conclustion:
easy algorithm issue, do not forget caculate the max_len with len at the end of program, such as the case [1,0,0,1,1,0,1,1,1,0,1,1,1,1,1]
